sklearn/manifold/spectral_embedding_.py

```diff
@@ -308,7 +317,7 @@ class SpectralEmbedding(BaseEstimator):
     Forms an affinity matrix given by the specified function and
     applies spectral decomposition to the corresponding graph laplacian.
     The resulting transformation is given by the value of the
-    eigenvectors for each data point.
+    u for each data point.
```

Comment:
The result of the embedding is given by the coefficients of the eigenvectors? I don't know ;)